config/crd/bases/v1/datadoghq.com_datadogagents.yaml

```diff
@@ -1284,6 +1284,108 @@ spec:
                             Default: true
                           type: boolean
                       type: object
+                    otelCollector:
+                      description: OtelCollector configuration.
+                      properties:
+                        conf:
```

Comment:
rename to config?

Reply:
Same as: https://github.com/DataDog/datadog-operator/pull/1559#discussion_r1886798434